src/frontend/Player/createPlayer.ts

```diff
@@ -10,5 +11,10 @@ export const createPlayer: VideoPlayerCreator = (
   switch (type) {
     case 'plyr':
       return createPlyrPlayer(ref, dispatchPlayerTimeUpdate, video);
+    case 'videojs':
+      const player = createVideojsPlayer(ref, dispatchPlayerTimeUpdate, video);
+      return {
+        destroy: () => player.dispose(),
+      };
```

Comment:
Should be good to add a default case which throws and reports an error.
In this way, Sentry is able to report mistake about configuration.